libft/tests/strings/ft_split_skip_delim_test.c

```diff
@@ -28,6 +28,47 @@ typedef struct s_string_array
 	size_t		count;
 }	t_strings;
 
+static void	free_tokens(char **tokens)
```

Comment:
we have a [bunch](https://github.com/ioannnikulin/minishell/blob/dev/libft/sources/memory/ft_free_s.c) of fancy ``free`` functions in libft, can we use them here?